addons/sourcemod/scripting/ttt/ttt_teleporter.sp

```diff
@@ -101,22 +104,30 @@ public Action TTT_OnItemPurchased(int client, const char[] itemshort, bool count
         {
             int role = TTT_GetClientRole(client);
```

Comment:
role is now unused: https://github.com/Bara/TroubleinTerroristTown/commit/0c9ca2e844bde2575af0834970234746d93ebcd7/checks?check_suite_id=270107673#step:5:614

I would like to see the role check back, because of the instant role item.